src/lib/testPaymentExpiry.js

```diff
@@ -100,40 +99,12 @@ TransferTester.prototype.validateExpiryHoldTime = function (expiresAt) {
   }
 }
 
-// We also need to check if we have enough time between the expiry
-// of the destination transfer with the latest expiry and the expiry of
-// the source transfer with the earliest expiry is greater than the
-// minMessageWindow.
-// This is done to ensure that we have enough time after the last
-// moment one of the destination transfers could happen (taking money out
-// of our account) to execute all of the source transfers
-TransferTester.prototype.validateMinMessageWindow = function () {
```

Comment:
(for the record we may want to express the "destinationExpiryDuration" as two dates instead of a duration: `destinationPrepareBy` and `destinationExpiresAt` or something like that)